_posts/2024-12-19-ols-nigeria-meetup.md

```diff
@@ -0,0 +1,49 @@
+---
+layout: post
+title: OLS Nigeria 
+authors:
+- npdebs
+- jilaga
+- seunolufemi123
+- tajuddeen1
+image: /images/ols-ng-meetup/ols-ng-group.jpeg
+photos:
+  name: Jilaga
+  license: CC BY-SA 4.0
+  url: 
+---
+
+# [Title to be decided]
+
+There’s something magical about meeting people you’ve worked with virtually for ages and instantly feeling like family when you finally see each other in real life. That’s exactly how it felt when **the OLS Nigeria Team met up for the first time** this past Tuesday, December 17, 2024, at the vibrant Art Tech District in **Abuja**.
+
+Let’s set the scene: four of us, coming from different parts of the country, converging in one place. [Taj]({% link people.md %}#tajuddeen1), our Program Coordinator for the Catalyst project, flew in from Kano; [Jilaga]({% link people.md %}#jilaga), the creative force behind OLS’s branding, joined us from Lagos; [Debs]({% link people.md %}#npdebs), our Web Developer and Community Manager extraordinaire, made her way from Port Harcourt; and [Seun]({% link people.md %}#seunolufemi123), our Abuja-based Resident Fellow, welcomed us to his city.
+
+By noon, we were all gathered, a mix of excitement and slight nervousness in the air—but that didn’t last long. It only took a few minutes of epic banter to break the ice: Debs and Taj debated whether the Art Tech District was really just a 10-minute walk from the hotel, with Seun chiming in to say he doesn’t trust Google Maps. (Spoiler: there’s a hilarious reason Debs and Taj couldn’t agree on the distance—read on!)
+
+Needless to say, we clicked. Big time.
+
+We kicked off our day with an **escape room challenge**—“a team-building exercise,” we told ourselves. The mission? Solve a murder case within 30 minutes. Did we crack it? Nope. (Turns out we’re better at our jobs than catching imaginary killers.) But we uncovered bigger mysteries – Taj is great at Chemistry, Jilaga’s an undercover detective, and Seun will seize every opportunity to taunt Debs by fumbling with the light switch.
+
+Next stop: the **Discovery Museum**. It was a walk through Nigerian history, culture, and creativity. We soaked up every moment, from admiring the exhibits to singing along to iconic Nigerian throwback tunes.  A documentary on the effect of the "Boko Haram" insurgency had us reflecting deeply on the resilience of our nation, and feeling grateful for the gift of life. And, of course, we couldn’t resist snapping photos at every turn…
+
+![image alt]({% link images/ols-ng-meetup/ols-ng-collage.jpg %}) 
```

Comment:
but what's in the image? :) Alt text please :)

Reply:
All in due time, dear Yochannah.

Reply:
Done, done, done! ✨